Update README.md

flipswitchingmonkey opened the pull request:
Added some shortcuts

flipswitchingmonkey committed:
Update README.md

fabiospampinato commented:
I think it would be best to wrap each of these embedded customizations in a `details` tag, for better UX, like so:

`````
<details>
<summary>My Title</summary>

```
My Customization
```

</details>
`````

> Insert text at beginning of line, e.g. headers

Calling the `editor.cursor.undo` command once or twice should move the cursor back to its original position, you'd probably want to use that too.

flipswitchingmonkey committed:
Update README.md

wrapped code blocks into details tags
note the extra blank line before code fence

flipswitchingmonkey commented:
Updated. editor.cursor.undo is not doing anything it seems, so I left it out for now.

fabiospampinato commented:
> editor.cursor.undo is not doing anything it seems, so I left it out for now.

Yeah, strange 🤔 I tried it in vscode too and it works he same way, so guess this command works differently then I would have expected.
